pvlib/tests/iotools/test_srml.py

```diff
@@ -74,19 +76,33 @@ def test__map_columns(column, expected):
     assert srml._map_columns(column) == expected
 
 
+@pytest.mark.remote_data
+@pytest.mark.flaky(reruns=RERUNS, reruns_delay=RERUNS_DELAY)
+def test_get_srml():
+    url = 'http://solardat.uoregon.edu/download/Archive/EUPO1801.txt'
+    file_data = srml.read_srml(url)
+    requested, _ = srml.get_srml(station='EU', start='2018-01-01',
+                                 end='2018-01-31')
+    assert file_data.equals(requested)
```

Comment:
Nitpick: it is somewhat better to use `assert_frame_equal` so that if it the assertion eventually fails, more useful diagnostic information (like which column is different, e.g.) is reported.